middleware/message.js

```diff
@@ -0,0 +1,17 @@
+const setMessage = (req, res, next) => {
+    //The value of the message is set into res.locals.message so that it is available via the variable 
```

Comment:
Good design & great comments! This is super helpful and it's always best to explain the _why_ behind code when it's not clear as you have done here.